Update MPSL and SoftDevice Controller (#891)
nrfconnect/sdk-nrfxlib: softdevice_controller/CHANGELOG.rst

   :local:
   :depth: 2

> All the notable changes to this project are documented in this file.

Reviewer: ```suggestion
All the notable changes to this project are documented in this page.
```

Main branch
***********

All the notable changes included in the main branch are documented in this section.

Bug fixes
=========

> * Fixed an issue where the controller was using a non-zero randomness for the very first advertising event even after calling :c:func:`sdc_hci_cmd_vs_set_adv_randomness` with a valid ``adv_handle`` parameter (DRGN-18261).

Reviewer 2: ```suggestion
* Fixed an issue where the controller was using non-zero randomness for the first advertising event.
  This happened even after calling :c:func:`sdc_hci_cmd_vs_set_adv_randomness` with a valid ``adv_handle`` parameter (DRGN-18261).
```
Split to two sentences to increase readability.
Randomness is not countable. Alternatively, it could "using a non-zero random delay" or something along those lines if I've understood the function properly.

> * Fixed an issue where the controller would end up in the HardFault handler after receiving an invalid response to a scan-request (DRGN-18358).

Reviewer 2: ```suggestion
* Fixed an issue where the controller would end up in the HardFault handler after receiving an invalid response to a scan request (DRGN-18358).
```
